src/main/java/com/exadel/discountwebapp/role/service/RoleService.java

```diff
@@ -25,7 +25,7 @@
     @Transactional(propagation = Propagation.REQUIRED, readOnly = true)
     public RoleResponseVO getRoleById(Long id) {
         Optional<Role> role = roleRepository.findById(id);
```

Comment:
Let's write it in the same way we do it in other places:
Role role = roleRepository.findById(id)
                                          .orElseThrow(...);